src/corona_hakab_model/Parameters/simulation_parameters_example.py

```diff
@@ -1,3 +1,5 @@
+from util import get_numpy_uniform_dist
```

Comment:
no need to import

Reply:
I think this all file is a joke, we should remove all this files

Reply:
i agree... but the problem is that tom and shir are using the config files.
so it's easier for them to copy new values from this file

Reply:
I get it, but maybe we should auto-generate it instead of filling everytime (i.e. having a command/scrpit that generates them)

Reply:
Also it is called .py that why the import was needed, it should be .json

Reply:
I agree that generating them and making them be json is correct